Dexter/eng 1205 otel traces broke during the rename (#68)
humanlayer/agentcontrolplane: CLAUDE.md

   ```go
   // Good: Only update Phase when transitioning to a new workflow stage
   // while preserving current Status (health)

>    trtc.Status.Phase = acp.ToolCallaitingHumanApproval

Reviewer: LEGIT

2. **Change Status Only When Health State Changes**: Status should change only when the health or readiness of the resource changes:
   ```go
   // When a resource encounters an error

>    trtc.Status.Status = acp.ToolCallypeError

Reviewer: ```suggestion
   trtc.Status.Status = acp.ToolCallTypeError
```

   trtc.Status.Error = err.Error()
   
   // When a resource becomes ready

>    trtc.Status.Status = acp.ToolCallypeReady

Reviewer: ```suggestion
   trtc.Status.Status = acp.ToolCallTypeReady
```

4. **Early Return for Terminal States**: For resources that have workflow-based lifecycles with terminal states, add an early check in the Reconcile function to avoid unnecessary processing:
   ```go

>    // For workflow-based resources like ToolCalleach terminal states

Reviewer: ```suggestion
   // For workflow-based resources like ToolCall that reach terminal states
```
